tools/auto-compose/LICENSE

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
```

Comment:
Remove this file. 

Please add a note to README citing the author of dag-factory and calling out that dag-factory is MIT Licensed.

All code that you have written and plan to open source in this repo should have and apache 2.0 header and is covered under out top level license file in this repo.